src/main/java/frc/utils/applicationsutils/CMDHandler.java

```diff
@@ -23,15 +25,25 @@ public static void runCMDCommand(Path directory, String command) {
     public static void runCMDCommand(String command) {
         String cmdSpecification = isWindows() ? WINDOWS_CMD_SPECIFICATION : NON_WINDOWS_CMD_SPECIFICATION;
```

Comment:
```suggestion
        String osShell = isWindows() ? WINDOWS_CMD_SPECIFICATION : NON_WINDOWS_CMD_SPECIFICATION;
```

Reply:
`os`? what is `os`? did you mean`operatingSystem`?

Reply:
Its known name so i think it's ok to use it, but you can use operatingSystem instead